code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -154,8 +137,40 @@
 				belly += 1
 				StageChange(TRUE)
 
+/mob/living/simple_animal/hostile/abnormality/mountain/attacked_by(obj/item/I, mob/living/user)
+	if(LAZYLEN(patrol_path) && CanAttack(user)) // Basically, it will retaliate while patrolling without stopping or chasing after them
+		user.attack_animal(src)
```

Comment:
My main concern here is that this would retaliate at the same speed as the attacker hits, so Torn Off Wings would be hit VERY quickly and disproportionally punish fast attackers

Reply:
During testing this never came up.